src/table/column.jsx

```diff
@@ -35,6 +35,10 @@ export default class Column extends React.Component {
          * 单元格的对齐方式
          */
         align: PropTypes.oneOf(['left', 'center', 'right']),
+        /**
+         * 单元格标题的对齐方式
```

Comment:
Would you add some explanation in document, telling users that **if not specified, use the same as 'align' **